public/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Mac files
+.DS_Store
```

Comment:
**nitpick**: it should be possible to define this in the `.gitignore` at the root.